open_schools_platform/user_management/users/migrations/0002_firebasetoken.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.2.12 on 2022-10-26 14:27
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import rules.contrib.models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FirebaseToken',
+            fields=[
+                ('created_at', models.DateTimeField(db_index=True, default=django.utils.timezone.now)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('id', models.UUIDField(default=uuid.uuid4, primary_key=True, serialize=False)),
+                ('token', models.CharField(blank=True, max_length=200, null=True)),
```

Comment:
firebase_token

Reply:
But if we refer to this field it would be something like "user.firebase_token.firebase_token", is it ok?

I named this field just "token" for more convenient referring to it. What do you think?

Reply:
hmm, good question. I think I need to add logic for checking, whether provided firebase token is valid or not.